tfp-collab-userspace-domain/DomainObjects/GalleryDo.cs

```diff
@@ -0,0 +1,9 @@
+namespace tfp_collab_userspace_domain.DomainObjects;
+
+public class GalleryDo
+{
+    public GalleryId Id { get; set; }
+    public string Name { get; set; }
```

Comment:
You should suppress warnings lik this  public string Name { get; set; } = null!;
There a several other occurences on this.